src/header/__tests__/header.test.tsx

```diff
@@ -69,3 +69,13 @@ test('supports title tag name override with non-default variant', () => {
   expect(wrapper.find('h3')).toBeNull();
   expect(wrapper.find('h2')!.getElement()).toHaveTextContent('title');
 });
+
+test('adds the correct data attribute', () => {
```

Comment:
minor: not a great name for a test (applies to some of the others in this PR, too). I would except this to atleast mention "analytics" somehow, and probably have it sit in its own analytics suite (even if its just one test case)